event_sourcery-postgres.gemspec

```diff
@@ -24,7 +24,7 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency 'sequel', '~> 4.38'
   spec.add_dependency 'pg'
-  spec.add_dependency 'event_sourcery', '>= 0.10.0'
+  spec.add_dependency 'event_sourcery', '>= 0.14.0'
```

Comment:
We're making use of the `Event#with` method from `event_sourcery` which is only available since version 0.14. Make this dependency explicit.